Model/ResourceModel/Syncs.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace Klaviyo\Reclaim\Model\ResourceModel;
+
+class Syncs extends \Magento\Framework\Model\ResourceModel\Db\AbstractDb
+{
+    const SYNCED = 'SYNCED';
+
+    /**
+     * Define main table
+     */
+    protected function _construct()
+    {
+        $this->_init('kl_sync', 'id');
+    }
+
+    public function updateRowsToSynced($ids)
+    {
+        if (empty($ids)) {
+            return;
+        }
```

Comment:
This logic is being repeated all over the place. Is it necessary or is there a way to do this better? can we check in the callers maybe?